cmd/namespace/namespace.go

```diff
@@ -44,6 +44,10 @@ func Namespace(ctx context.Context) *cobra.Command {
 				return err
 			}
 
+			if !okteto.IsOktetoCluster() {
+				return errors.ErrNotOktetoCluster
+			}
+
 			err := RunNamespace(ctx, namespace)
```

Comment:
This is also deprecated by `okteto kubeconfig`. We could also call the `Kubeconfig` logic from here